README.md

```diff
@@ -77,11 +79,22 @@ on how to correct them.
 ## How to run Nammu
 
 To run Nammu on your computer, you can download it from
-[here](https://github.com/oracc/nammu/releases/download/0.5.0/nammu-0.5.5.jar)
+[here](https://github.com/oracc/nammu/releases/download/0.6/nammu-0.6.0.jar)
 and just double click on it to open it and use it.
 
-If you find any problem trying to open it, it might be that you don't have Java
-installed. You can check if you do by opening a terminal and typing:
+If you find any problem trying to open it,
```

Comment:
Well spotted, I forgot to add "have a look at the Troubleshooting section" and link to it.